src/Card/Card.component.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react"
+import styled, { css } from "styled-components"
+import CardBody, { CardBodyProps } from "./CardBody.component"
+import CardFooter, { CardFooterProps } from "./CardFooter.component"
+import CardHeader, { CardHeaderProps } from "./CardHeader.component"
+import CardImage, { ImageProps } from "./CardImage.component"
+import CardFlipper, { CardFlipperProps } from "./CardFlipper.component"
+
+interface CardComposition {
+  Body: React.FC<CardBodyProps>
+  Header: React.FC<CardHeaderProps>
+  Footer: React.FC<CardFooterProps>
+  Image: React.FC<ImageProps>
+  Flipper: React.FC<CardFlipperProps>
+}
+
+export interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Card will expand across available width
+   * @default false
+   */
+  fluid?: boolean
+  /**
+   * Raises the card, applying a small box-shadow
+   * @default false
+   */
+  raised?: boolean
+  /**
+   * Main image to be displayed
+   *
+   */
+  image?: string
+  /**
+   * Translate Y on hover
+   * @default false
+   */
+  hoverable?: boolean
+}
+
+const CardContainer = styled.div<CardProps>`

```

Comment:
For example, this should go on the styles file. 

You might want to look into using the extending the box into styled instead of using a `div`.  It would be something like `styled(Box)`. `Box` already has a lot of methods from `styled-system` thus you wouldn't need to add that into the `styled-component`. For instance, because it extends `Box` you can jus call `<Card flex="content" flexDirection="column" padding="auto margin"/>` just as an example. If you want we can go more in depth with this point on slack. Feel free to reach out :)